lab3/client/src/views/story-book.js

```diff
@@ -0,0 +1,26 @@
+export default function (store) {
+	return class StoryBookComponent extends window.HTMLElement {
+		constructor () {
+			super();
+			this.store = store;
+			// TODO: initial DOM rendering of story itself
+			this.innerHTML = '<p id=story> Here Goes A Line Of Story </p>';
```

Comment:
should display a list of story with their description rather than just a single static html.